libs/shared/interfaces/src/lib/common/value.interface.ts

```diff
@@ -4,4 +4,6 @@ export interface Value {
   label: string
   value: string
   icon?: ReactNode
+  isDisabled?: boolean
```

Comment:
Ahah I also had to add a property here... a `description` property for things like radio box where we have a value and a description